modules/sdk-coin-sol/src/sol.ts

```diff
@@ -714,17 +721,40 @@ export class Sol extends BaseCoin {
 
     const completedTransaction = await txBuilder.build();
     const serializedTx = completedTransaction.toBroadcastFormat();
+    const derivationPath = `m/${scanIndex}`;
+    const inputs = [
+      {
+        address: completedTransaction.inputs[0].address,
+        valueString: completedTransaction.inputs[0].value,
+        value: new BigNumber(completedTransaction.inputs[0].value).toNumber(),
+      },
+    ];
+    const outputs = [
+      {
+        address: completedTransaction.outputs[0].address,
+        valueString: completedTransaction.inputs[0].value,
```

Comment:
I know that this is what was specified, but kinda curious why we want value and valueString for inputs but not for outputs

Reply:
yeah i just followed exactly what ovc specified, let me follow up with Islam

Reply:
just keeping it as is for now